plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/editors/EditorContentFunction.java

```diff
@@ -127,7 +135,7 @@ public void setText(String text) {
         this.text = text;
     }
 
-    public String getProjectName() {
+    private String getProjectName() {
```

Comment:
Any specific reason to introduce this private method? Since this method is only being used inside the class itself and it only returns the projectName variable value, we can directly use the variable without going through a method to reduce the number of code lines. We introduce methods to reduce the number of code lines and avoid code repetitions where applicable. :)